spec/Gaufrette/Adapter/FlysystemSpec.php

```diff
@@ -31,11 +31,46 @@ function it_reads_file(AdapterInterface $adapter)
         $this->read('filename')->shouldReturn('Hello.');
     }
 
+    function it_throws_file_not_found_exception_when_trying_to_read_an_unexisting_file(AdapterInterface $adapter)
+    {
+        $adapter->read('filename')->willThrow(new FileNotFoundException('filename'));
+
+        $this->shouldThrow('Gaufrette\Exception\FileNotFound')->duringread('filename');
```

Comment:
It looks surprising but yes, it's because of an old version of phpspec (2.4). BTW I ran into issues when trying to run the phpspec tests : the [`phpspec.yml`](https://github.com/KnpLabs/Gaufrette/blob/master/phpspec.yml) file has a dependency on [`Akeneo\SkipExampleExtension`](https://github.com/akeneo/PhpSpecSkipExampleExtension/blob/master/src/Akeneo/SkipExampleExtension.php) but I couldn't get it installed via composer (unresolvable set of packages). So I had to comment that file to run the tests on my machine :confused: . I'm still wondering how the tests can be executed on the CI .

It would be a good point to bump up the tests tools to their latest version :wink: